src/main/webapp/jsp/studentCourseDetails.jsp

```diff
@@ -9,7 +9,7 @@
     <script type="text/javascript" src="/js/student.js"></script>
 </c:set>
 
-<ts:studentPage bodyTitle="Team Details for ${data.studentCourseDetailsPanel.courseId}" pageTitle="TEAMMATES - Student">
+<ts:studentPage bodyTitle="Team Details for ${data.studentCourseDetailsPanel.courseId}" pageTitle="TEAMMATES - Student" jsIncludes="${jsIncludes}">
```

Comment:
This is a little dubious, it worked without the JavaScript? If it worked without, do we really need it?